CHANGELOG.md

```diff
@@ -109,6 +109,10 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `config-schema.json` now allows an array of strings or nested table for the
   `ui.pager` setting.
 
+* Divergent changes are no longer marked red in immutable revisions. Since the
+  revision is immutable, the user shouldn't take any action, so the red color
+  was unnecessarily alarming.
+
```

Comment:
I would recommend that the [docs](https://jj-vcs.github.io/jj/latest/FAQ/#how-do-i-deal-with-divergent-changes-after-the-change-id) be updated to let the user know that no action should be taken in this case

Reply:
This was taken care of in #7061